agent-inject/agent/agent.go

```diff
@@ -521,6 +533,34 @@ func serviceaccount(pod *corev1.Pod) (string, string) {
 	return serviceAccountName, serviceAccountPath
 }
 
+// IRSA support - get aws_iam_token volume mount details to inject to vault containers
+func getAwsIamTokenAccount(pod *corev1.Pod) (string, string) {
```

Comment:
How about renaming this to getAwsIamTokenVolume?

```suggestion
func getAwsIamTokenVolume(pod *corev1.Pod) (string, string) {
```